code/code/misc/player_data.cc

```diff
@@ -2277,6 +2277,39 @@ std::vector<sstring> listAccountCharacters(sstring name)
   return list;
 }
 
+int count_set_bit(int n) {
+   int count = 0;
+   while(n != 0) {
+      if((n & 1) == 1) {
+         count++;
+      }
+      n = n >> 1; //right shift 1 bit
+   }
+   return count;
+}
+
+sstring parseClass(unsigned short code){
```

Comment:
Don't we have something similar in the `who` command?

Reply:
We do yes: https://github.com/sneezymud/sneezymud/blob/master/code/code/misc/multiclass.cc#L350

Unfortunately it's a member of TBeing and we don't have any being objects here only codes that were read from the charfile for the account.

Reply:
Since you didn't put it into a `::namespace` (nor marked as `static`, perhaps we could then call this code from `TBeing::getProfAbbrevName()`? It feels creepy to have the same thing twice.

Reply:
I don't like it either. Let me take a look and see what we can do.